### ISSUE TITLE
sta::write_liberty crash

### ISSUE THREAD

**Issue by maliberty** [parallaxsw/OpenSTA#48]
I realize this is an internal function but it is helpful and it would be nice if it didn't crash on CCS libraries.

You can reproduce it in ORFS with
```
read_liberty platforms/asap7/lib/CCS/asap7sc7p5t_INVBUF_RVT_FF_ccs_220122.lib.gz
sta::write_liberty asap7sc7p5t_INVBUF_RVT_FF_ccs_211120 dump.lib
```

Fwiw I worked around this with 
```
class TableAxis
{
...
  size_t size() const { return values_ ? values_->size() : 0; }
```

**Comment by maliberty** [comment]
The stack
```
 2# sta::LibertyWriter::writeTableAxis4(sta::TableAxis const*, int) in openroad
 3# sta::LibertyWriter::writeTableTemplate(sta::TableTemplate const*) in openroad
 4# sta::LibertyWriter::writeLibrary() in openroad
 5# sta::writeLiberty(sta::LibertyLibrary*, char const*, sta::StaState*) in openroad
```

**Comment by jjcherry56** [comment]
fixed by d4afed2a sta::write_liberty table template with no Index values
